net_util/src/queue_pair.rs

```diff
@@ -69,6 +69,8 @@ impl TxVirtio {
                             iov_len: desc.len() as libc::size_t,
                         };
                         iovecs.push(iovec);
+                    } else {
+                        return Err(NetQueuePairError::DescriptorChainInvalid);
```

Comment:
Do we want to  print an error message with the value of `desc.len` and `desc.is_write_only()` before we return? It can be helpful to understand the error when the error occurs.